test/mocks.py

```diff
@@ -153,15 +154,15 @@ def _get_object_from_file(filename: str) -> dict[str, Any]:
 
 
 def set_up_ok_mocks(
-    rqm,
+    requests_mock,
```

Comment:
use consistent name to refer to the requests mock (totally not because I did a search-n-replace of `rqm`)